R/pkg/NAMESPACE

```diff
@@ -247,3 +247,5 @@ export("structField",
        "structType.jobj",
        "structType.structField",
        "print.structType")
+
+export("as.data.frame")
```

Comment:
is there a reason why this isn't on the same list of export for DataFrame above?